packages/radio-button/src/RadioButton.test.js

```diff
@@ -0,0 +1,62 @@
+import { mount } from "enzyme";
+import React from "react";
+import renderer from "react-test-renderer";
+import RadioButton from "./RadioButton";
+
+describe("RadioButton", () => {
+  describe("integration", () => {
+    it("renders correctly", () => {
+      const tree = renderer
+        .create(<RadioButton label="HIG RadioButton" />)
+        .toJSON();
+
+      expect(tree).toMatchSnapshot();
+    });
+  });
+
+  describe("event handlers", () => {
+    let eventHandler;
+    const subject = (props = {}) =>
+      mount(<RadioButton title="RadioButton" {...props} />);
+    let wrapper;
+
+    beforeEach(() => {
+      eventHandler = jest.fn();
```

Comment:
Copying over Morris' [comment](https://github.com/Autodesk/hig/pull/1053#discussion_r205197253) from my work, which I think is also relevant here:

> Nitpick: This only needs to be called once. __We can either put it in a before() block, or just move it outside and use const eventHandler = jest.fn();

Reply:
Sure, updated in f2e52bd4. Quick note that was copied over from an existing test when we started Checkbox.  (I believe from https://github.com/Autodesk/hig/blob/d7571620a72951c4eb98fb644ba75f6719f18b76/packages/button/src/Button.test.js#L14) Could consider addressing more widely